autoload/lsp/internal/diagnostics/signs.vim

```diff
@@ -127,15 +127,13 @@ function! s:set_signs(params) abort
 endfunction
 
 function! s:place_signs(server, diagnostics_response, bufnr) abort
+    let l:linecount = get(getbufinfo(a:bufnr)[0],'linecount',0) " linecount available from vim 8.2.0019
```

Comment:
I would update the version here so it it doesn't match we don't even enable the flag.
 https://github.com/prabirshrestha/vim-lsp/blob/7d59166b18699d28a44c0c66d4299272ca26d62c/autoload/lsp/utils.vim#L16-L19

Reply:
you can then update the docs. https://github.com/prabirshrestha/vim-lsp/blob/7d59166b18699d28a44c0c66d4299272ca26d62c/doc/vim-lsp.txt#L608-L610